src/index-debug.html

```diff
@@ -24,38 +24,17 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <title>Photoshop Design Space (optimized build)</title>
-    <link rel="stylesheet" type="text/css" href="style/main.css" />
-    <script src="js/require.js"></script>
-    <script>
-        var locale = window.navigator.language,
-            parts = locale.match(/\w+/),
-            language = parts.length > 0 && parts[0];
+    <title>Photoshop Design Space (development build)</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <script type ="text/javascript" src="./externalDeps.js"></script>
```

Comment:
This is bad because it's loading one bundle after another, which is slower than just loading everything at once. Look at this profile run:
![image](https://cloud.githubusercontent.com/assets/1075154/11513309/33c4401c-9827-11e5-8f9b-b290ed398925.png)
It takes over 900ms before we start executing our own JavaScript. This seems like it must be a performance regression compared to using the sources compiled by r.js...